cmd/edit/machinepool/nodepool.go

```diff
@@ -86,6 +89,14 @@ func editNodePool(cmd *cobra.Command, nodePoolID string, clusterKey string, clus
 	if isTaintsSet || interactive.Enabled() {
 		npBuilder = npBuilder.Taints(taintBuilders...)
 	}
+	var awsNodePoolBuilder *cmv1.AWSNodePoolBuilder
+	if isTagsSet || interactive.Enabled() {
+		awsNodePoolBuilder = cmv1.NewAWSNodePool().Tags(awsTags)
```

Comment:
@gdbranco Did we confirm with the BU this is what they want? My understanding from the last few conversations on this is that it will cause all nodes in the nodepool to be recreated and we're deliberately blocking that for other operations IIUC.

Reply:
No, I'll be keeping tags as hidden and remove the interactive though